### PR TITLE
style front: fix typos & punctuation

### PR DESCRIPTION
**Pull request by Noisy96**
# Description
This PR fixes various typos, missed punctuation and letter case.

## Type of change
Minor typos fixes

## Remarks
* I took the liberty to include fixes in articles descriptions (which I suppose would usually require original author's review) because I believe the changes did not affect the meaning and only dealt with language.

### CODE REVIEW

File: frontend/src/apps/main/pages/landing/what-and-why/index.tsx [hunk 1]
```diff
@@ -7,12 +7,13 @@ const reasons = [
...
     description:
-      "Centralizing the issues around Algerian problems brings care and commitment from all , and you are an example of that",
+      "Centralizing the issues around Algerian problems brings care and commitment from all, and you are an example of that.",
   },
   {
     title: "Local Contributions",
     description:
-      "Bringing your project to spotlight and let local collaborators find you, which will lead to an even better version of you library.",
+      /* This still does not make sense to me, what do you mean by "library"? */
+      "Bringing your project to the spotlight and letting local collaborators find you, which will lead to an even better version of your library.",
```
Comment (ZibanPirate) [inline frontend/src/apps/main/pages/landing/what-and-why/index.tsx:16]: yes i agree with you, i think "software" would better better fitted word, what do u think?
```suggestion
      "Bringing your project to the spotlight and letting local collaborators find you, which will lead to an even better version of your software.",
```

**Review by ZibanPirate — CHANGES_REQUESTED** [review]
Thank you @Noisy96 for the changes, i opened one thread about the "library" word, i suggested to use "software" instead as it's a more general term.

File: frontend/src/apps/main/pages/landing/what-and-why/index.tsx (hunk 1 above)
Comment (Noisy96) [reply frontend/src/apps/main/pages/landing/what-and-why/index.tsx:16]: Yes, perfect :thumbsup:
it even covers datasets, according to some defintions [1](https://en.wikipedia.org/wiki/Software) [2](https://searchapparchitecture.techtarget.com/definition/software)

**Review by ZibanPirate — APPROVED** [review]
Solid work @Noisy96 , thanks for the fixes!

**Pull request merged**